keyboards/qwertykeys/qk65/r2/readme.md

```diff
@@ -0,0 +1,25 @@
+# QK65 - Hotswap R2 PCB
+
+![qk65](https://i.imgur.com/kFhx5qS.png)
+
+A 65% made by Qwertykeys.
+
+* Keyboard Maintainer: [Selwyn](https://github.com/Siilwyn)
+* Hardware Supported: QK65 hotswap R2 PCB
+
+Make example for this keyboard (after setting up your build environment):
+
+    make qwertykeys/qk65/r2:default
+
+Flashing example for this keyboard:
+
+    make qwertykeys/qk65/r2:default:flash
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+
+## Bootloader
+
+Enter the bootloader in two ways:
+
+* **Bootmagic reset**: Hold down the B key and plug in the keyboard
```

Comment:
This is actually not the bootmagic reset (the old “full bootmagic” code did that, but that code had been removed from QMK some time ago).  If the “hold down the B key” method actually works on the board flashed with this QMK firmware, that method is actually implemented by the bootloader.

Also apparently the bootmagic reset does not work on this PCB (and neither does `QK_BOOT`), because the bootloader is not actually `stm32duino`, and the method to communicate with the bootloader is not actually known?  Unfortunately, the code from https://github.com/owlab-git/qmk_firmware/tree/qk65/keyboards/qwertykeys/qk65/r2 does not offer anything better (it also pretends that the bootloader is `stm32duino`).

Reply:
> If the “hold down the B key” method actually works on the board flashed with this QMK firmware, that method is actually implemented by the bootloader.

Yes this does work, should I call this by a different name? Maybe "Bootloader reset"?

Sadly `QK_BOOT` indeed does not work, shall I mention this in the readme?